gamemode/modules/police/sv_init.lua

```diff
@@ -107,12 +107,12 @@ function plyMeta:arrest(time, arrester)
     end
 end
 
-function plyMeta:unArrest(unarrester)
+function plyMeta:unArrest(unarrester, teleportOverride)
```

Comment:
This adds a boolean argument that cannot really be interpreted from the call site without a comment:

```lua
ply:unArrest(someOtherPly, true)
```

In the above snippet, it's hard to interpret the `true`. I think there might be a better solution.

Reply:
I see your point with that. I can't really think of a better solution for this.